src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp

```diff
@@ -317,3 +515,7 @@ template struct GeneralizedHarmonic::ComputeDuDt<3>;
 template struct GeneralizedHarmonic::ComputeNormalDotFluxes<1>;
 template struct GeneralizedHarmonic::ComputeNormalDotFluxes<2>;
 template struct GeneralizedHarmonic::ComputeNormalDotFluxes<3>;
+
```

Comment:
Please switch this to use `GENERATE_INSTANTIATIONS` given how many things are now being instantiated.